### DISCUSSION TITLE
Error: cts.tcl, 86 couldn't execute "eqy": no such file or directory

### DISCUSSION THREAD

**Discussion by ReemAbuRajab7** [The-OpenROAD-Project/OpenROAD#6056] [Q&A]
![Screenshot from 2024-10-29 23-54-26](https://github.com/user-attachments/assets/f2c0250d-fca7-404f-b41e-87584c318576)
![WhatsApp Image 2024-10-29 at 11 58 07 PM (1)](https://github.com/user-attachments/assets/7bffdf8f-3f69-4424-b04a-be787b5dd0ca)

to fix it , I found this file ' 4_eqy_test.eqy ' and export it's path, but doesn't work  .

**Comment by maliberty** [comment]
You need to have eqy installed.  If you use the dependency installer it should take care of it.

**Reply by rovinski** [reply]
Additionally, you can simply turn of equivalency checking (unset `EQUIVALENCE_CHECK` in your design config).

**Reply by aboke1988** [reply]
I have my all dependencies installed but still getting same error. I tried turn of equivalency checking (unset EQUIVALENCE_CHECK) and the run is going through properly.

But i have to proceed the run with (EQUIVALENCE_CHECK) . What are the dependencies need to be installed so that "eqy" can run smoothly?

**Reply by maliberty** [reply]
The message suggests eqy is not installed 
![image](https://github.com/user-attachments/assets/f5cd13f5-b7e4-4f14-9069-6f7f518c4630)
